sonar-ballerina-plugin/src/main/resources/io/ballerina/sonar/ballerina-4.html

```diff
@@ -0,0 +1,36 @@
+<p>
+    Class methods can be isolated. An isolated method is same as an isolated function with self treated as a parameter.
+    An isolated method call is concurrency-safe if both the object is safe and the arguments are safe.
+    When isolated keyword is not used, the method might not be safe in a concurrent environment.
```

Comment:
This rule is defined for public constructs only

Reply:
Resolved with f10b04b7337bff186e2de3ae2a65a2421c10884d